src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
```

Comment:
make 3 as a constant

Reply:
The checkstyle doesn't allow using constant naming here,

[ERROR] C:\Users\andre\IdeaProjects\jv-lottery\src\main\java\core\basesyntax\Application.java:5:19: Abbreviation in name 'LOOP_COUNT' must contain no more than '2' consecutive capital letters. [AbbreviationAsWordInName]

But will do as you say

Reply:
Ok, i figured it out. Constants are static final fields and belong to a class, not a variable or method.